src/parser/core/modules/Procs.tsx

```diff
@@ -288,12 +296,15 @@ export abstract class Procs extends Analyser {
 	private checkConsumeProc(procGroup: ProcGroup, event: Events['action']): boolean {
 		// If we don't currently have this proc active, we can't possibly consume it
 		if (!this.currentWindows.has(procGroup)) { return false }
+		const action = this.data.getAction(event.action)
 		// If we have no idea what this action is, it doesn't consume it
-		if (this.data.getAction(event.action) == null) { return false }
+		if (action == null) { return false }
+		// If we're in downtime, we don't consume procs (don't credit using an AoE-around-self Proc during downtime)
+		if (this.downtime.isDowntime(event.timestamp)) { return false }
```

Comment:
I am wondering if there are any cases where jobs might want to override this and still flag procs dropped during downtime, for example healer abilities. The only ones I can think of are things like Liturgy of the Bell or Macrocosmos which just execute their effect when they expire, but it might be worth asking in healer channel if there's a use case for exposing an "ignore downtime" flag which defaults to true?

Reply:
I don't know that Liturgy or Macro would really be something to track with Procs anyway, since they don't require re-activation to gain their effect, just give an early activation of their on-expires effect if you do reactivate.

The intent with this PR is to be 100% consistent with the behavior of the current module, and then take a look at potential changes in future PRs if necessary.

Reply:
Yea that makes sense! I mentioned it because I think White Mage now gets a proc-like thing off Temperance and I'm not sure if it's possible for that to be wasted on expiration, but it's definitely something that could be added later.

Reply:
Doesn't sound like there currently are, please disregard this comment :)